### ISSUE TITLE
Unofficial servers "Version mismatch!"

### ISSUE THREAD

**Issue by aChtourou** [Uuvana-Studios/longvinter-windows-client#1181]
**Describe the bug**
G-Portal servers (and other unofficial servers) for LongVinter say "Version mismatch!". When trying to join, the player is returned to the main menu after a period of time. First experienced on Friday (8/19).

**To Reproduce**
1. From the main menu select Play to open the servers list and then select the Unofficial Servers tab.
2. Try to join a server hosted by G-Portal (to see this information, click the 3 dot expandable on the right side), or any server that says "Version mismatch!"
3. Be returned to the main menu after being unable to connect to the server.

**Expected behavior**
To be able to connect to an unofficial server.

**Screenshots**
![20220822181358_1](https://user-images.githubusercontent.com/19596050/186029352-9fc9ed0b-f008-4ca4-9a79-13664f0a85cf.jpg)

**Desktop**
 - OS: Windows
 - Game Version: 1.0.8b
 - Steam Version: 1660960119

**Additional context**
- I think this might be caused by G-Portal not yet receiving the most recent update or that somehow there's a problem with the update that I received.
- I verified game file integrity on Steam.
- As per G-Portal support, "If the owner of the server has 'Auto-update' enabled, the server will get updated as soon as Gportal gets the update."
- The server that I'm trying to connect to has Auto-update activated through G-Portal. We've verified this and have also tried restarting it multiple times.
- I've tried connecting to our household owned G-Portal server as well as to others, with the same results.

**Comment by faketucker** [comment]
It seems like you're on version 1.0.8b and it's not the experimental branch, so this is the G-portal problem. You should contact their support to update their servers instead.

**Comment by faketucker** [comment]
There was also no update on 8.19 the last update was 23 days ago.

**Comment by aChtourou** [comment]
Oh! Sorry that I misunderstood--on Steam, the LongVinter post from Friday says, "Regular Update, Friday DevLog #22", and I mistook what that meant (I thought it was an update with the wipe and that's what the dev log was regarding). That being said, is it possible that G-Portal has the experimental version installed on their servers mistakenly? If I understand correctly, the experimental update was released to experimental servers on Friday. Maybe G-Portal got that and something went wrong there? I contacted G-Portal prior to posting here and they said to check with the game about it.
